eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -182,13 +184,16 @@ instance EraPParams era => ToCBOR (ConwayGovPredFailure era) where
 instance EraPParams era => FromCBOR (ConwayGovPredFailure era) where
   fromCBOR = fromEraCBOR @era
 
+data ConwayGovEvent era 
+  = GovNewProposalsSnapshot !(TxId (EraCrypto era)) !(ProposalsSnapshot era) 
```

Comment:
Let's call it `GovNewProposals`, since `ProposalsSnapshot` will be soon renamed to `Proposals` too.
```suggestion
data ConwayGovEvent era 
  = GovNewProposals !(TxId (EraCrypto era)) !(ProposalsSnapshot era) 
```